cfgen/cfgen

```diff
@@ -1115,6 +1116,19 @@ class ConfProcess(ToDhall):
                     assert ctrl is not None
                     ConfDrive.build(m0conf, ctrl,
                                     ConfSdev.build(m0conf, svc_id, disk))
+
+            if stype is SvcT.M0_CST_CAS:
+                assert proc_desc is not None
+                if meta_data:
```

Comment:
just one minor change, add an else condition and set disk to `/dev/null` if metadata is not present in CDF.

```
            if stype is SvcT.M0_CST_CAS:
                assert proc_desc is not None
                if meta_data:
                    disk = proc_desc['io_disks'].get('meta_data')
                    assert ctrl is not None
                else:
                    disk = '/dev/null'
                ConfDrive.build(m0conf, ctrl,
                                ConfSdev.build(
                                    m0conf, svc_id,
                                    Disk(
                                        path=disk,
                                        size=1024,
                                        blksize=1)))
```

Reply:
Addressed.